app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -14,8 +14,15 @@ def default_home_feed(user_signed_in: false)
         stories
       end
 
-      def featured_story_and_default_home_feed(user_signed_in: false, ranking: true)
-        featured_story, hot_stories = globally_hot_articles(user_signed_in)
+      # @param user_signed_in [Boolean] are we treating this as an
+      #        anonymous user?
```

Comment:
What is an anonymous user? Is it a user that has not signed in?

Reply:
I really don't know…I'm following the logic of the code.  Sometimes we always initialize this object with `user: current_user` but when the controller says `user_singed_in? == false` we go down this path.

So…it's a weird misnomer in the code that I'm not prepared to tackle at this point.